client/assets/images/stats/calypso-purchase-stats-graph.svg

```diff
@@ -0,0 +1,178 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none">
+    <defs>
+        <filter id="shadow" x="-14.101" y="0.899" width="484.066" height="381.202"
+            filterUnits="userSpaceOnUse" color-interpolation-filters="sRGB">
+            <feFlood flood-opacity="0" result="BackgroundImageFix" />
+            <feColorMatrix in="SourceAlpha" values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 127 0" result="hardAlpha" />
+            <feOffset />
+            <feGaussianBlur stdDeviation="27.05" />
+            <feColorMatrix values="0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0 0.08 0" />
+            <feBlend in2="BackgroundImageFix" result="effect1_dropShadow_3568_12212" />
+            <feBlend in="SourceGraphic" in2="effect1_dropShadow_3568_12212" result="shape" />
+        </filter>
+        <symbol id="stats" width="456" height="383" viewBox="0 0 456 383" fill="none">
+            <g filter="url(#shadow)" >
+                <rect  width="375.864" height="273" x="40" y="55" fill="#fff" rx="12.27" />
+            </g>
+            <g>
+                <rect width="375.864" height="273" x="40" y="55" fill="#fff" rx="12.27" />
+                <path
+                    fill="#101517"
+                    fill-rule="evenodd"
+                    d="M91.053 101.218a2.687 2.687 0 1 0 0-5.374 2.687 2.687 0 0 0 0 5.374Zm0-1.612a1.075 1.075 0 1 0
+    0-2.15 1.075 1.075 0 0 0 0 2.15Z"
+                    clip-rule="evenodd"
+                />
+                <path
+                    fill="#101517"
+                    d="M88.635 107.667v-2.15a2.956 2.956 0 0 0-2.956-2.956h-4.3a2.956 2.956 0 0 0-2.955
+    2.956v2.15h1.612v-2.15c0-.742.602-1.344 1.343-1.344h4.3c.742 0 1.343.602 1.343
+    1.344v2.15h1.612ZM96.158
+    105.517v2.15h-1.612v-2.15c0-.742-.602-1.344-1.344-1.344h-2.687v-1.612h2.687a2.956 2.956 0 0 1 2.956
+    2.956Z"
+                />
+                <path
+                    fill="#101517"
```

Comment:
It's a shame we can't only change the fill value instead of copying the whole file, but I think it's not possible with the value being so deep in the document structure.

Reply:
It looks like we need to refactor the graph to be fillable with those color schemes rather than generating a bunch of files.